src/itype.h

```diff
@@ -342,13 +342,19 @@ struct islot_wheel
 
         /** width of wheel (inches) */
         int width = 0;
+
+        /** coefficient of friction (percent) */
+        int friction = 100;
 };
 
 struct islot_fuel
 {
     public:
         /** Energy of the fuel (kilojoules per charge) */
         float energy = 0.0f;
+
+        /** Air to fuel ratio.  */
+        float ratio = 1.0f;
```

Comment:
Don't add values you don't intend to use in the same PR, unless it's a lot of code and you need to split things up.
Keep it for a PR that actually uses the values.